utils/are_you_sure.go

```diff
@@ -0,0 +1,72 @@
+package utils
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+	"syscall"
+
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+type UserInputReader interface {
+	Read(secret bool) (string, error)
+}
+
+type StdinInputReader struct{}
+
+func NewStdinInputReader() *StdinInputReader {
+	return &StdinInputReader{}
+}
+
+func (reader StdinInputReader) Read(secret bool) (string, error) {
+	if secret {
+		bytePassword, err := terminal.ReadPassword(int(syscall.Stdin))
+		if err != nil {
+			return "", err
+		}
+		response := string(bytePassword)
+		return response, nil
+
+	} else {
+		reader := bufio.NewReader(os.Stdin)
+		response, err := reader.ReadString('\n')
+		if err != nil {
+			return "", err
+		}
+		return response, nil
+	}
+}
+
+func AreYouSure(message string, reader UserInputReader) bool {
+	for {
+		res := Ask(fmt.Sprintf("%s [y/n]", message), false, false, reader)
+		if res == "y" {
+			return true
+		} else if res == "n" {
+			return false
+		}
```

Comment:
I think you can do something like this.
```suggestion
		res := Ask(fmt.Sprintf("%s [y/n]", message), false, false, reader)
	        return res == "y"
```

Reply:
You are right, this was just code I moved but looks more concise this way.